arkane/molpro.py

```diff
@@ -265,8 +267,6 @@ def loadConformer(self, symmetry=None, spinMultiplicity=0, opticalIsomers=None,
 
         # Close file when finished
         f.close()
-        if opticalIsomers is None:
```

Comment:
Maybe merge this and the `Get loadConformer to automatically estimate optical isomers` commits instead of deleting?

Reply:
I don't really understand what two commits you'd like me to merge.

Reply:
commit https://github.com/ReactionMechanismGenerator/RMG-Py/pull/1571/commits/dfe4c606e559b61be95121874dd56ea1884ba707 added the lines
```python
if opticalIsomers is None:
    opticalIsomers = self.get_optical_isomers_and_symmetry_number()[0]
```
and commit https://github.com/ReactionMechanismGenerator/RMG-Py/pull/1571/commits/7f7a91e213920d67d94bba30f35f40bd02bf19cd removes them.
Maybe merge these two, or don't add the lines to begin with if they're deleted here?

Reply:
Oh yeah. My goal behind that was to separate commits which make small changes that each commit can function properly on their own, which is why there were lines added and removed. I'll combine these two commits as you suggested.

Reply:
Just merged those two commits with the commit between them, which removed `symfromlog` parameter.